apk-generator/v2/app/generator/generator.py

```diff
@@ -221,6 +221,7 @@ def download_event_data(self):
         self.save_file_in_temp_assets('speakers')
         self.save_file_in_temp_assets('sponsors')
         self.save_file_in_temp_assets('tracks')
+	self.save_file_in_temp_assets('session_types')
```

Comment:
@arp95 here endpoint is 'sessions/types'.
https://eventyay.com/api/v1/events/6/sessions/types
http://open-event-dev.herokuapp.com/api/v1/#!/sessions/get_session_type_list